### ISSUE TITLE
Character creator

### ISSUE THREAD

**Issue by hops678** [Psych-Slice/P-Slice#71]
### What feature do you want to get added on the **P-Slice** engine?

Can you add a Character select screen editor where you can make and edit characters on the screen

**Comment by moxie-coder** [comment]
> ### What feature do you want to get added on the **P-Slice** engine?
> Can you add a Character select screen editor where you can make and edit characters on the screen

dude psych already has that, just press 7 on the main menu

**Comment by ThatJanWTM** [comment]
> dude psych already has that, just press 7 on the main menu

I think they mean like the actual Character Select Screen not the actual Character Editor

**Comment by hops678** [comment]
yes exactly that's what it needs a character select screen editor
